components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -256,6 +264,38 @@ class AutoPushFeature(
         service.deleteToken()
     }
 
+    /**
+     * Verifies status (active, expired) of the push subscriptions and then notifies observers.
+     */
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun verifyActiveSubscriptions() {
+        DeliveryManager.with(connection) {
```

Comment:
talked offline: rename `with` to `ifInitialized`

Reply:
Filed: https://github.com/mozilla-mobile/android-components/issues/4822

Contributor PR will follow soon behind. :)